c31-pryhaven/nutshell2
# Av chat get messages

# Description

#5.1 (17) Chat Messages. Messages stored in the API now appear when clicking the "messages" tab in the nav bar.

Applies to issue # 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

```git fetch --all```
```git checkout av-chat-get-messages```
run react-app
click "messages" tab
confirm that messages appear on the dom.


# Checklist:
